cmd/expiration-mailer/main_test.go

```diff
@@ -516,15 +516,18 @@ func insertCertificate(cert certDERWithRegID, lastNagSent time.Time) error {
 		return fmt.Errorf("inserting certificate: %w", err)
 	}
 
-	return setupDBMap.Insert(ctx, &core.CertificateStatus{
-		Serial:                core.SerialToString(parsedCert.SerialNumber),
-		LastExpirationNagSent: lastNagSent,
-		Status:                core.OCSPStatusGood,
-		NotAfter:              parsedCert.NotAfter,
-		OCSPLastUpdated:       time.Time{},
-		RevokedDate:           time.Time{},
-		RevokedReason:         0,
-	})
+	_, err = setupDBMap.ExecContext(context.Background(),
+		`INSERT INTO certificateStatus
+		(serial, notAfter, status, ocspLastUpdated, revokedDate, revokedReason, lastExpirationNagSent)
+		VALUES (?, ?, ?, ?, ?, ?, ?)`,
```

Comment:
Should we be using the non-questionmark, more mysql-friendly, placeholder here?

Reply:
Question marks work for both mysql and mariadb; Postgres uses $1, $2, etc. gorm offers named `:placeholder` syntax to go with a map of names -> values. So far we use both in various places and I think haven't made a decision to go 100% with one or the other. For this test I'm inclined to do the easy thing and leave it as-is because it will very soon be deleted.